homeassistant/components/websocket_api/commands.py

```diff
@@ -275,9 +275,15 @@ def _template_listener(event, updates):
             )
         )
 
-    info = async_track_template_result(
-        hass, [TrackTemplate(template, variables)], _template_listener
-    )
+    try:
+        info = async_track_template_result(
+            hass,
+            [TrackTemplate(template, variables)],
+            _template_listener,
+            raise_on_template_error=True,
+        )
+    except TemplateError as ex:
+        raise vol.Invalid(f"invalid template ({ex})") from ex
```

Comment:
Can we raise something else here so we can catch these errors in the connection an suppress them or log them with lower than error?
Currently it's quite spammy while testing templates

Reply:
With this change you'll get one line in the log instead of the whole backtrace.  I'll work on suppressing the logging

Reply:
I've removed the logging and updated the test to ensure we explicitly do not log and the error goes to the websocket instead.